sdk/samples/iot/paho_iot_provisioning_sample.c

```diff
@@ -220,30 +223,31 @@ static void register_device_with_provisioning_service(void)
   pubmsg.retained = 0;
 
   // Publish the register request.
-  if ((rc = MQTTClient_publishMessage(mqtt_client, register_publish_topic_buffer, &pubmsg, NULL))
+  if ((rc = MQTTClient_publishMessage(mqtt_client, register_topic_buffer, &pubmsg, NULL))
       != MQTTCLIENT_SUCCESS)
   {
-    IOT_SAMPLE_LOG_ERROR("Failed to publish register request: MQTTClient return code %d.", rc);
+    IOT_SAMPLE_LOG_ERROR("Failed to publish Register request: MQTTClient return code %d.", rc);
     exit(rc);
   }
 }
 
-static void receive_device_registration_status(void)
+static void receive_device_registration_status_message(void)
 {
   int rc;
   char* topic = NULL;
   int topic_len = 0;
   MQTTClient_message* message = NULL;
-  az_iot_provisioning_client_register_response register_response;
-  az_iot_provisioning_client_operation_status operation_status;
   bool is_operation_complete = false;
 
   // Continue to parse incoming responses from the provisioning pervice until the device
```

Comment:
```suggestion
  // Continue to parse incoming responses from the provisioning service until the device
```
```suggestion
  // Continue to parse incoming responses from the provisioning pervice until the device
```

Reply:
i need to get that spell checker running... lol